set.mm

```diff
@@ -489692,6 +489775,74 @@ of the indicator function (Contributed by Thierry Arnoux,
       HUDIAJDUEUOSUFADUGUHUIAUTUQPABCDHIGUJUKAVAURUSFACDHIULUMUN $.
   $}
 
+$(
+  @{
+    reprdifc.c @e |- C = { c e. ( A ( repr ` S ) M ) | -. ( c ` x ) e. B } @.
+    reprdifc.a @e |- ( ph -> A C_ NN ) @.
+    reprdifc.m @e |- ( ph -> M e. NN0 ) @.
+    reprdifc.s @e |- ( ph -> S e. NN0 ) @.
+    @( Express the representations as a sum of integers in a difference of sets
+       using conditions on each of the indices.  (Contributed by Thierry
+       Arnoux, 27-Dec-2021.) @)
+    reprdifc @p |- ( ph -> ( A ( repr ` S ) M ) = ( ( B ( repr ` S ) M )
+      u. U_ x e. ( 0 ..^ S ) C ) ) @=
+      ? @.
+  @}
+
+  @{
+    reprpmtr.o @e |- O = { c e. ( A ( repr ` S ) M ) | -. ( c ` 0 ) e. B } @.
+    reprpmtr.p @e |- P = { c e. ( A ( repr ` S ) M ) | -. ( c ` X ) e. B } @.
+    reprpmtr.t @e |- T = ( ( pmTrsp ` ( 0 ..^ S ) ) ` { 0 , X } ) @.
+    reprpmtr.f @e |- F = ( c e. P |-> ( c o. T ) ) @.
+    reprpmtr.x @e |- ( ph -> X e. ( 0 ..^ S ) ) @.
+    @( Transposing ` 0 ` and ` X ` maps representations with a condition on the
+       first index to transpositions with the same condition on the index ` X `
+       .  (Contributed by Thierry Arnoux, 27-Dec-2021.) @)
+    reprpmtrf1o @p |- ( ph -> F : P -1-1-onto-> O ) @=
+      ? @.
+  @}
+$)
+
+  ${
+    $d N n $.  $d i n $.
+    $( Value of the second Chebyshev function, or summary von Mangoldt
+       function.  (Contributed by Thierry Arnoux, 28-Dec-2021.) $)
+    chpvalz $p |- ( N e. ZZ -> ( psi ` N ) = sum_ n e. ( 1 ... N ) ( Lam ` n )
```

Comment:
"summary von Mangoldt function" -> "the summatory function of the von Mangoldt function" (see Wikipedia)